### Getting all arxiv pdfs listed in the following Github repo:: https://github.com/georgestein/ml-in-cosmology

### Then using https://pypi.org/project/arxiv-dl/

In [1]:
import re
# from tqdm import tqdm

def read_markdown_file(file_path):
    """Read the content of a Markdown file and return it."""
    with open(file_path, 'r', encoding='utf-8') as file:
        content = file.read()
    return content

def extract_arxiv_links(markdown_content):
    """Extract sections and arXiv links from the markdown content."""
    pattern = re.compile(r"##\s*(.*?)\n(.*?)##", re.DOTALL)
    section_links = {}
    for match in pattern.finditer(markdown_content + "##"):  # Appending '##' to ensure last section is captured
        section_title = match.group(1).strip()
        section_content = match.group(2)
        links = re.findall(r"https://arxiv.org/abs/\d+\.\d+", section_content)
        section_links[section_title] = links
    return section_links

# Path to the github repo Markdown file
file_path = 'github_repo.md'

# Reading the markdown file
markdown_content = read_markdown_file(file_path)

# Extracting arXiv links
links_by_section = extract_arxiv_links(markdown_content)

# Printing the links by section
for section, links in links_by_section.items():
    print(f"### {section}")
    for link in links:
        print(link)
    print("\n")


### ml-in-cosmology


### ## Dictionary
https://arxiv.org/abs/1502.03509
https://arxiv.org/abs/1310.1659
https://arxiv.org/abs/1706.04008
https://arxiv.org/abs/1606.04797


### ## Structure Formation
https://arxiv.org/abs/1303.1055
https://arxiv.org/abs/1409.1576
https://arxiv.org/abs/1203.1695
https://arxiv.org/abs/1312.2101
https://arxiv.org/abs/1510.06402
https://arxiv.org/abs/1510.07659
https://arxiv.org/abs/1711.02033
https://arxiv.org/abs/1712.03255
https://arxiv.org/abs/1712.04467
https://arxiv.org/abs/1801.09070
https://arxiv.org/abs/1802.04271
https://arxiv.org/abs/1803.11156
https://arxiv.org/abs/1804.00816
https://arxiv.org/abs/1805.02247
https://arxiv.org/abs/1805.04537
https://arxiv.org/abs/1811.06533
https://arxiv.org/abs/1901.01264
https://arxiv.org/abs/1902.05965
https://arxiv.org/abs/1903.10524
https://arxiv.org/abs/1903.12173
https://arxiv.org/abs/1904.12846
https://arxiv.org/abs/1905.06958
https://arxiv.org/abs/1906.06339
https://arxiv.org/abs/1908.05519
https://arxi

In [3]:
links_by_section.keys()

dict_keys(['ml-in-cosmology', '## Dictionary', '## Structure Formation', 'Structure Identification', 'Gravitational Lensing', 'Weak Lensing', 'Strong Lensing', 'Observational', 'Redshifts', 'Other Observational', 'Tools', 'Reviews'])

### For a first pass, let us pick the Gravitational Lensing papers and download them with https://pypi.org/project/arxiv-dl/

In [14]:
def get_links_for_section(links_by_section, section_name):
    """
    Extracts and returns the list of arXiv links for a specified section.
    
    Parameters:
    links_by_section (dict): A dictionary where keys are section names and values are lists of arXiv links.
    section_name (str): The name of the section from which to extract arXiv links.
    
    Returns:
    list: A list of arXiv links for the specified section. Returns an empty list if the section is not found.
    """
    # Return the links for the specified section, or an empty list if the section is not found
    return links_by_section.get(section_name, [])


In [16]:
section_to_extract = "Strong Lensing"
extracted_links = get_links_for_section(links_by_section, section_to_extract)

strong_lensing_links = []

# Print the extracted links
print(f"ArXiv links for section '{section_to_extract}':")
for link in extracted_links:
    strong_lensing_links.append(link)
    print(link)

ArXiv links for section 'Strong Lensing':
https://arxiv.org/abs/1607.04644
https://arxiv.org/abs/1703.02642
https://arxiv.org/abs/1704.02322
https://arxiv.org/abs/1704.02744
https://arxiv.org/abs/1708.08842
https://arxiv.org/abs/1708.08843
https://arxiv.org/abs/1802.03609
https://arxiv.org/abs/1807.04764
https://arxiv.org/abs/1808.00011
https://arxiv.org/abs/1901.01359
https://arxiv.org/abs/1906.00970
https://arxiv.org/abs/1909.02005
https://arxiv.org/abs/1909.07346
https://arxiv.org/abs/1910.00015
https://arxiv.org/abs/1910.06157
https://arxiv.org/abs/1911.04320
https://arxiv.org/abs/1911.03867
https://arxiv.org/abs/2004.13048
https://arxiv.org/abs/2005.04730
https://arxiv.org/abs/2005.05353
https://arxiv.org/abs/2007.00188
https://arxiv.org/abs/2008.12731
https://arxiv.org/abs/2009.06639
https://arxiv.org/abs/2009.06663
https://arxiv.org/abs/2010.12960
https://arxiv.org/abs/2010.07032
https://arxiv.org/abs/2012.00042
https://arxiv.org/abs/2109.00014
https://arxiv.org/abs/2109.09781
h

In [17]:
len(strong_lensing_links)

47

In [18]:
section_to_extract = "Weak Lensing"
extracted_links = get_links_for_section(links_by_section, section_to_extract)

weak_lensing_links = []

# Print the extracted links
print(f"ArXiv links for section '{section_to_extract}':")
for link in extracted_links:
    weak_lensing_links.append(link)
    print(link)

ArXiv links for section 'Weak Lensing':
https://arxiv.org/abs/1002.0838
https://arxiv.org/abs/1411.3193
https://arxiv.org/abs/1706.02390
https://arxiv.org/abs/1707.05167
https://arxiv.org/abs/1802.01212
https://arxiv.org/abs/1806.05995
https://arxiv.org/abs/1807.02120
https://arxiv.org/abs/1807.08732
https://arxiv.org/abs/1808.07491
https://arxiv.org/abs/1810.11027
https://arxiv.org/abs/1810.11030
https://arxiv.org/abs/1812.05781
https://arxiv.org/abs/1902.03663
https://arxiv.org/abs/1902.08161
https://arxiv.org/abs/1906.03156
https://arxiv.org/abs/1908.00543
https://arxiv.org/abs/1911.12890
https://arxiv.org/abs/2005.06551
https://arxiv.org/abs/2006.07011
https://arxiv.org/abs/2007.06529
https://arxiv.org/abs/2011.08271
https://arxiv.org/abs/2011.10438
https://arxiv.org/abs/2109.11060
https://arxiv.org/abs/2201.05561
https://arxiv.org/abs/2203.06124
['https://arxiv.org/abs/1607.04644', 'https://arxiv.org/abs/1703.02642', 'https://arxiv.org/abs/1704.02322', 'https://arxiv.org/abs/1704.

In [19]:
len(weak_lensing_links)

26

In [21]:
### There is a nested list inside weak_links (why??)

### Make one list::

import itertools

# Use itertools.chain to flatten the list
flat_list = list(itertools.chain.from_iterable([sublist if isinstance(sublist, list) else [sublist] for sublist in weak_lensing_links]))

In [22]:
flat_list

['https://arxiv.org/abs/1002.0838',
 'https://arxiv.org/abs/1411.3193',
 'https://arxiv.org/abs/1706.02390',
 'https://arxiv.org/abs/1707.05167',
 'https://arxiv.org/abs/1802.01212',
 'https://arxiv.org/abs/1806.05995',
 'https://arxiv.org/abs/1807.02120',
 'https://arxiv.org/abs/1807.08732',
 'https://arxiv.org/abs/1808.07491',
 'https://arxiv.org/abs/1810.11027',
 'https://arxiv.org/abs/1810.11030',
 'https://arxiv.org/abs/1812.05781',
 'https://arxiv.org/abs/1902.03663',
 'https://arxiv.org/abs/1902.08161',
 'https://arxiv.org/abs/1906.03156',
 'https://arxiv.org/abs/1908.00543',
 'https://arxiv.org/abs/1911.12890',
 'https://arxiv.org/abs/2005.06551',
 'https://arxiv.org/abs/2006.07011',
 'https://arxiv.org/abs/2007.06529',
 'https://arxiv.org/abs/2011.08271',
 'https://arxiv.org/abs/2011.10438',
 'https://arxiv.org/abs/2109.11060',
 'https://arxiv.org/abs/2201.05561',
 'https://arxiv.org/abs/2203.06124',
 'https://arxiv.org/abs/1607.04644',
 'https://arxiv.org/abs/1703.02642',
 'h

In [23]:
total_lensing_lst = strong_lensing_links + flat_list
len(total_lensing_lst)

119

### Now try and download these 119 arxiv papers on Lensing

In [24]:
grav_lensing_arxiv = total_lensing_lst

In [32]:
import subprocess
import os

def download_paper(url, download_dir):
    """
    Downloads a paper from the given URL using the 'paper' command-line tool.
    
    Parameters:
        url (str): The URL to download the paper from.
        download_dir (str): The directory to download the paper into.
    """
    # Construct the command for a single paper download
    command = [
        'paper',  # Command name
        '-p',  # Download PDFs only
        # '-d', download_dir,  # Set the download directory
        url  # The paper URL to download
    ]

    # Execute the command
    result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
    # Optionally print output and errors (or handle them as needed)
    print("Output:", result.stdout.decode())
    print("Error:", result.stderr.decode())

In [33]:
def download_papers(paper_urls):
    """
    Downloads each paper from the given list of URLs individually using the 'paper' command-line tool.
    
    Parameters:
        paper_urls (list): A list of URLs to download.
    """
    # Create download directory name based on the list name
    download_dir = f"{paper_urls}_papers"
    
    # Ensure the directory exists
    if not os.path.exists(download_dir):
        os.makedirs(download_dir)

    # Loop through each URL and download it
    for url in paper_urls:
        download_paper(url, download_dir)

In [34]:
len(grav_lensing_arxiv)

119

In [35]:
# grav_lensing_arxiv

### Using the default download directory (~/Downloads for now)

In [43]:
def download_paper(url):
    """
    Downloads a paper from the given URL using the 'paper' command-line tool.
    
    Parameters:
        url (str): The URL to download the paper from.
    """
    # Construct the command for a single paper download
    command = [
        'paper',  # Command name
        '-p',  # Download PDFs only
         url  # The paper URL to download
    ]

    # Execute the command
    result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
    # Optionally print output and errors (or handle them as needed)
    print("Output:", result.stdout.decode())
    print("Error:", result.stderr.decode())

In [44]:
grav_lensing_arxiv[0]

'https://arxiv.org/abs/1607.04644'

In [46]:
from tqdm import tqdm

In [47]:
for url in tqdm(grav_lensing_arxiv):
    download_paper(url)

  0%|          | 0/119 [00:00<?, ?it/s]

  1%|          | 1/119 [00:08<15:56,  8.10s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1607.04644


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1607.04644_A_Neural_Network_Gravitational_Arc_Finder_based_on_the_Mediatrix_filamentation_Method.pdf"



  2%|▏         | 2/119 [00:15<15:25,  7.91s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1703.02642


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1703.02642_CMU_DeepLens_Deep_Learning_For_Automatic_Image-based_Galaxy-Galaxy_Strong_Lens_Finding.pdf"



  3%|▎         | 3/119 [00:24<16:11,  8.38s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1704.02322


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1704.02322_Automated_Lensing_Learner_Automated_Strong_Lensing_Identification_with_a_Computer_Vision_Technique.pdf"



  3%|▎         | 4/119 [00:33<16:08,  8.43s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1704.02744


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1704.02744_Finding_strong_lenses_in_CFHTLS_using_convolutional_neural_networks.pdf"



  4%|▍         | 5/119 [00:41<15:30,  8.17s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1708.08842


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1708.08842_Fast_Automated_Analysis_of_Strong_Gravitational_Lenses_with_Convolutional_Neural_Networks.pdf"



  5%|▌         | 6/119 [00:47<14:35,  7.75s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1708.08843


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1708.08843_Uncertainties_in_Parameters_Estimated_with_Neural_Networks_Application_to_Strong_Gravitational_Lensing.pdf"



  6%|▌         | 7/119 [00:56<14:50,  7.95s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1802.03609


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1802.03609_The_Strong_Gravitational_Lens_Finding_Challenge.pdf"



  7%|▋         | 8/119 [01:04<14:56,  8.08s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1807.04764


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1807.04764_Testing_Convolutional_Neural_Networks_for_finding_strong_gravitational_lenses_in_KiDS.pdf"



  8%|▊         | 9/119 [01:12<14:47,  8.07s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1808.00011


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1808.00011_Analyzing_interferometric_observations_of_strong_gravitational_lenses_with_recurrent_and_convolutional_neural_networks.pdf"



  8%|▊         | 10/119 [01:21<14:47,  8.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1901.01359


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1901.01359_Data-Driven_Reconstruction_of_Gravitationally_Lensed_Galaxies_using_Recurrent_Inference_Machines.pdf"



  9%|▉         | 11/119 [01:31<15:59,  8.89s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1906.00970


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1906.00970_Finding_Strong_Gravitational_Lenses_in_the_DESI_DECam_Legacy_Survey.pdf"



 10%|█         | 12/119 [01:39<15:28,  8.68s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1909.02005


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1909.02005_Mining_for_Dark_Matter_Substructure_Inferring_subhalo_population_properties_from_strong_lenses_with_machine_learning.pdf"



 11%|█         | 13/119 [01:47<14:49,  8.40s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1909.07346


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1909.07346_Deep_Learning_the_Morphology_of_Dark_Matter_Substructure.pdf"



 12%|█▏        | 14/119 [01:55<14:19,  8.19s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1910.00015


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1910.00015_Direct_Detection_of_Dark_Matter_Substructure_in_Strong_Lens_Images_with_Convolutional_Neural_Networks.pdf"



 13%|█▎        | 15/119 [02:03<14:22,  8.30s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1910.06157


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1910.06157_Differentiable_Strong_Lensing_Uniting_Gravity_and_Neural_Nets_through_Differentiable_Probabilistic_Programming.pdf"



 13%|█▎        | 16/119 [02:18<17:26, 10.16s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1911.04320


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1911.04320_Identifying_Strong_Lenses_with_Unsupervised_Machine_Learning_using_Convolutional_Autoencoder.pdf"



 14%|█▍        | 17/119 [02:37<21:43, 12.77s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1911.03867


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1911.03867_A_Modular_Deep_Learning_Pipeline_for_Galaxy-Scale_Strong_Gravitational_Lens_Detection_and_Modeling.pdf"



 15%|█▌        | 18/119 [02:48<20:56, 12.44s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2004.13048


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2004.13048_HOLISMOKES_--_II_Identifying_galaxy-scale_strong_gravitational_lenses_in_Pan-STARRS_using_convolutional_neural_networks.pdf"



 16%|█▌        | 19/119 [03:03<22:04, 13.24s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2005.04730


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2005.04730_Discovering_New_Strong_Gravitational_Lenses_in_the_DESI_Legacy_Imaging_Surveys.pdf"



 17%|█▋        | 20/119 [03:12<19:17, 11.69s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2005.05353


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2005.05353_Dark_Matter_Subhalos_Strong_Lensing_and_Machine_Learning.pdf"



 18%|█▊        | 21/119 [03:19<17:10, 10.52s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2007.00188


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2007.00188_Deep_Learning_for_Strong_Lensing_Search_Tests_of_the_Convolutional_Neural_Networks_and_New_Candidates_from_KiDS_DR3.pdf"



 18%|█▊        | 22/119 [03:27<15:41,  9.70s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2008.12731


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2008.12731_Decoding_Dark_Matter_Substructure_without_Supervision.pdf"



 19%|█▉        | 23/119 [03:35<14:43,  9.20s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2009.06639


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2009.06639_Extracting_the_Subhalo_Mass_Function_from_Strong_Lens_Images_with_Image_Segmentation.pdf"



 20%|██        | 24/119 [03:43<13:55,  8.80s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2009.06663


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2009.06663_Image_segmentation_for_analyzing_galaxy-galaxy_strong_lensing_systems.pdf"



 21%|██        | 25/119 [03:51<13:11,  8.42s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2010.12960


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2010.12960_Hunting_for_Dark_Matter_Subhalos_in_Strong_Gravitational_Lensing_with_Neural_Networks.pdf"



 22%|██▏       | 26/119 [03:58<12:42,  8.20s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2010.07032


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2010.07032_Targeted_Likelihood-Free_Inference_of_Dark_Matter_Substructure_in_Strongly-Lensed_Galaxies.pdf"



 23%|██▎       | 27/119 [04:07<12:45,  8.32s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2012.00042


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2012.00042_Large-Scale_Gravitational_Lens_Modeling_with_Bayesian_Neural_Networks_for_Accurate_and_Precise_Inference_of_the_Hubble_Constant.pdf"



 24%|██▎       | 28/119 [04:17<13:22,  8.82s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2109.00014


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2109.00014_Strong_lens_systems_search_in_the_Dark_Energy_Survey_using_Convolutional_Neural_Networks.pdf"



 24%|██▍       | 29/119 [04:27<14:01,  9.35s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2109.09781


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2109.09781_Finding_quadruply_imaged_quasars_with_machine_learning_I_Methods.pdf"



 25%|██▌       | 30/119 [04:46<18:06, 12.21s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.00023


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2110.00023_Mining_for_Strong_Gravitational_Lenses_with_Self-supervised_Learning.pdf"



 26%|██▌       | 31/119 [04:55<16:10, 11.03s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.01905


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2110.01905_High-quality_strong_lens_candidates_in_the_final_Kilo_Degree_survey_footprint.pdf"



 27%|██▋       | 32/119 [05:04<15:06, 10.42s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.02418


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2110.02418_The_DES_Bright_Arcs_Survey_Candidate_Strongly_Lensed_Galaxy_Systems_from_the_Dark_Energy_Survey_5000_Sq_Deg_Footprint.pdf"



 28%|██▊       | 33/119 [05:17<16:18, 11.38s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.09202


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2110.09202_Finding_Strong_Gravitational_Lenses_Through_Self-Attention.pdf"



 29%|██▊       | 34/119 [05:26<15:03, 10.63s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.11972


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2110.11972_Strong_lensing_in_UNIONS_Toward_a_pipeline_from_discovery_to_modeling.pdf"



 29%|██▉       | 35/119 [05:34<13:51,  9.90s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2112.02479


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2112.02479_Exploring_the_interpretability_of_deep_neural_networks_used_for_gravitational_lens_finding_with_a_sensitivity_probe.pdf"



 30%|███       | 36/119 [05:45<13:51, 10.02s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2202.12776


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2202.12776_Detecting_gravitational_lenses_using_machine_learning_exploring_interpretability_and_sensitivity_to_rare_lensing_configurations.pdf"



 31%|███       | 37/119 [05:53<13:03,  9.56s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2202.07663


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2202.07663_GIGA-Lens_Fast_Bayesian_Inference_for_Strong_Gravitational_Lens_Modeling.pdf"



 32%|███▏      | 38/119 [06:01<12:19,  9.12s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2203.09536


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2203.09536_Developing_a_Victorious_Strategy_to_the_Second_Strong_Gravitational_Lensing_Data_Challenge.pdf"



 33%|███▎      | 39/119 [06:09<11:49,  8.87s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2204.05924


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2204.05924_DeepZipper_II_Searching_for_Lensed_Supernovae_in_Dark_Energy_Survey_Data_with_Deep_Learning.pdf"



 34%|███▎      | 40/119 [06:26<14:35, 11.09s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2206.02764


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2206.02764_New_Strong_Gravitational_Lenses_from_the_DESI_Legacy_Imaging_Surveys_Data_Release_9.pdf"



 34%|███▍      | 41/119 [06:35<13:40, 10.51s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2205.09126


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2205.09126_Estimating_the_warm_dark_matter_mass_from_strong_lensing_images_with_truncated_marginal_neural_ratio_estimation.pdf"



 35%|███▌      | 42/119 [06:43<12:31,  9.76s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2206.11279


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2206.11279_HOLISMOKES_--_IX_Neural_network_inference_of_strong-lens_parameters_and_uncertainties_from_ground-based_images.pdf"



 36%|███▌      | 43/119 [06:51<11:46,  9.30s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2206.14820


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2206.14820_Strong_Lensing_Source_Reconstruction_Using_Continuous_Neural_Fields.pdf"



 37%|███▋      | 44/119 [06:59<11:13,  8.98s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.01073


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2207.01073_Pixelated_Reconstruction_of_Gravitational_Lenses_using_Recurrent_Inference_Machines.pdf"



 38%|███▊      | 45/119 [07:07<10:46,  8.74s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.04123


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2207.04123_Population-Level_Inference_of_Strong_Gravitational_Lenses_with_Neural_Network-Based_Selection_Correction.pdf"



 39%|███▊      | 46/119 [07:16<10:35,  8.70s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.10124


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2207.10124_HOLISMOKES_--_X_Comparison_between_neural_network_and_semi-automated_traditional_modeling_of_strong_lenses.pdf"



 39%|███▉      | 47/119 [07:37<14:56, 12.45s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.10698


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2207.10698_A_machine_learning_based_approach_to_gravitational_lens_identification_with_the_International_LOFAR_Telescope.pdf"



 40%|████      | 48/119 [07:45<12:53, 10.89s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1002.0838


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1002.0838_Bias-Free_Shear_Estimation_using_Artificial_Neural_Networks.pdf"



 41%|████      | 49/119 [07:52<11:37,  9.96s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1411.3193


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1411.3193_Hopfield_Neural_Network_deconvolution_for_weak_lensing_measurement.pdf"
Object 727 0 not defined.



 42%|████▏     | 50/119 [08:02<11:12,  9.75s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1706.02390


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1706.02390_CosmoGAN_creating_high-fidelity_weak_lensing_convergence_maps_using_Generative_Adversarial_Networks.pdf"



 43%|████▎     | 51/119 [08:10<10:25,  9.20s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1707.05167


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1707.05167_Cosmological_model_discrimination_with_Deep_Learning.pdf"



 44%|████▎     | 52/119 [08:17<09:43,  8.70s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1802.01212


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1802.01212_Non-Gaussian_information_from_weak_lensing_data_via_deep_learning.pdf"



 45%|████▍     | 53/119 [08:25<09:22,  8.53s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1806.05995


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1806.05995_An_improved_cosmological_parameter_inference_scheme_motivated_by_deep_learning.pdf"



 45%|████▌     | 54/119 [08:34<09:17,  8.58s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1807.02120


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1807.02120_Weak-lensing_shear_measurement_with_machine_learning_teaching_artificial_neural_networks_about_feature_noise.pdf"



 46%|████▌     | 55/119 [08:42<09:03,  8.50s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1807.08732


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1807.08732_Cosmological_constraints_from_noisy_convergence_maps_through_deep_learning.pdf"



 47%|████▋     | 56/119 [08:50<08:44,  8.33s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1808.07491


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1808.07491_Weak_lensing_shear_estimation_beyond_the_shape-noise_limit_a_machine_learning_approach.pdf"



 48%|████▊     | 57/119 [08:59<08:46,  8.49s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1810.11027


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1810.11027_On_the_dissection_of_degenerate_cosmologies_with_machine_learning.pdf"



 49%|████▊     | 58/119 [09:07<08:27,  8.33s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1810.11030


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1810.11030_Distinguishing_standard_and_modified_gravity_cosmologies_with_machine_learning.pdf"



 50%|████▉     | 59/119 [09:15<08:07,  8.12s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1812.05781


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1812.05781_Denoising_Weak_Lensing_Mass_Maps_with_Deep_Learning.pdf"



 50%|█████     | 60/119 [09:22<07:55,  8.05s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1902.03663


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1902.03663_Weak_lensing_cosmology_with_convolutional_neural_networks_on_noisy_data.pdf"



 51%|█████▏    | 61/119 [09:30<07:39,  7.92s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1902.08161


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1902.08161_Galaxy_shape_measurement_with_convolutional_neural_networks.pdf"



 52%|█████▏    | 62/119 [09:38<07:33,  7.96s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1906.03156


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1906.03156_Cosmological_constraints_with_deep_learning_from_KiDS-450_weak_lensing_maps.pdf"



 53%|█████▎    | 63/119 [09:46<07:22,  7.89s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1908.00543


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1908.00543_Deep_learning_dark_matter_map_reconstructions_from_DES_SV_weak_lensing_data.pdf"



 54%|█████▍    | 64/119 [09:54<07:15,  7.91s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1911.12890


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/1911.12890_Noise_reduction_for_weak_lensing_mass_mapping_An_application_of_generative_adversarial_networks_to_Subaru_Hyper_Suprime-Cam_first-year_data.pdf"



 55%|█████▍    | 65/119 [10:02<07:05,  7.88s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2005.06551


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2005.06551_Parameter_Inference_for_Weak_Lensing_using_Gaussian_Processes_and_MOPED.pdf"



 55%|█████▌    | 66/119 [10:10<07:00,  7.93s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2006.07011


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2006.07011_Shear_measurement_bias_II_a_fast_machine_learning_calibration_method.pdf"



 56%|█████▋    | 67/119 [10:17<06:46,  7.82s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2007.06529


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2007.06529_Interpreting_deep_learning_models_for_weak_lensing.pdf"



 57%|█████▋    | 68/119 [10:25<06:31,  7.68s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2011.08271


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2011.08271_Probabilistic_Mapping_of_Dark_Matter_by_Neural_Score_Matching.pdf"



 58%|█████▊    | 69/119 [10:32<06:20,  7.61s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2011.10438


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2011.10438_Higher_order_statistics_of_shear_field_a_machine_learning_approach.pdf"



 59%|█████▉    | 70/119 [10:39<06:10,  7.57s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2109.11060


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2109.11060_Simultaneously_constraining_cosmology_and_baryonic_physics_via_deep_learning_from_weak_lensing.pdf"



 60%|█████▉    | 71/119 [10:48<06:14,  7.81s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2201.05561


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2201.05561_Probabilistic_Mass_Mapping_with_Neural_Score_Estimation.pdf"



 61%|██████    | 72/119 [10:56<06:12,  7.92s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2203.06124


Error: [Processing] Retrieving paper metadata...
[Downloading] Using aria2 with 5 connections
[Done] Paper saved to "/Users/panchamb/Downloads/ArXiv_Papers/2203.06124_Accelerating_cosmological_inference_with_Gaussian_processes_and_neural_networks_--_an_application_to_LSST_Y1_weak_lensing_and_galaxy_clustering.pdf"



 61%|██████▏   | 73/119 [11:00<05:16,  6.87s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1607.04644


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1607.04644_A_Neural_Network_Gravitational_Arc_Finder_based_on_the_Mediatrix_filamentation_Method.pdf"



 62%|██████▏   | 74/119 [11:05<04:36,  6.15s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1703.02642


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1703.02642_CMU_DeepLens_Deep_Learning_For_Automatic_Image-based_Galaxy-Galaxy_Strong_Lens_Finding.pdf"



 63%|██████▎   | 75/119 [11:09<04:04,  5.55s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1704.02322


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1704.02322_Automated_Lensing_Learner_Automated_Strong_Lensing_Identification_with_a_Computer_Vision_Technique.pdf"



 64%|██████▍   | 76/119 [11:13<03:40,  5.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1704.02744


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1704.02744_Finding_strong_lenses_in_CFHTLS_using_convolutional_neural_networks.pdf"



 65%|██████▍   | 77/119 [11:17<03:23,  4.84s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1708.08842


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1708.08842_Fast_Automated_Analysis_of_Strong_Gravitational_Lenses_with_Convolutional_Neural_Networks.pdf"



 66%|██████▌   | 78/119 [11:21<03:09,  4.62s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1708.08843


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1708.08843_Uncertainties_in_Parameters_Estimated_with_Neural_Networks_Application_to_Strong_Gravitational_Lensing.pdf"



 66%|██████▋   | 79/119 [11:26<03:00,  4.52s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1802.03609


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1802.03609_The_Strong_Gravitational_Lens_Finding_Challenge.pdf"



 67%|██████▋   | 80/119 [11:30<02:51,  4.41s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1807.04764


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1807.04764_Testing_Convolutional_Neural_Networks_for_finding_strong_gravitational_lenses_in_KiDS.pdf"



 68%|██████▊   | 81/119 [11:34<02:44,  4.33s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1808.00011


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1808.00011_Analyzing_interferometric_observations_of_strong_gravitational_lenses_with_recurrent_and_convolutional_neural_networks.pdf"



 69%|██████▉   | 82/119 [11:38<02:38,  4.29s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1901.01359


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1901.01359_Data-Driven_Reconstruction_of_Gravitationally_Lensed_Galaxies_using_Recurrent_Inference_Machines.pdf"



 70%|██████▉   | 83/119 [11:42<02:33,  4.25s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1906.00970


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1906.00970_Finding_Strong_Gravitational_Lenses_in_the_DESI_DECam_Legacy_Survey.pdf"



 71%|███████   | 84/119 [11:47<02:28,  4.23s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1909.02005


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1909.02005_Mining_for_Dark_Matter_Substructure_Inferring_subhalo_population_properties_from_strong_lenses_with_machine_learning.pdf"



 71%|███████▏  | 85/119 [11:51<02:22,  4.20s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1909.07346


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1909.07346_Deep_Learning_the_Morphology_of_Dark_Matter_Substructure.pdf"



 72%|███████▏  | 86/119 [11:55<02:18,  4.19s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1910.00015


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1910.00015_Direct_Detection_of_Dark_Matter_Substructure_in_Strong_Lens_Images_with_Convolutional_Neural_Networks.pdf"



 73%|███████▎  | 87/119 [11:59<02:13,  4.18s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1910.06157


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1910.06157_Differentiable_Strong_Lensing_Uniting_Gravity_and_Neural_Nets_through_Differentiable_Probabilistic_Programming.pdf"



 74%|███████▍  | 88/119 [12:03<02:09,  4.16s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1911.04320


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1911.04320_Identifying_Strong_Lenses_with_Unsupervised_Machine_Learning_using_Convolutional_Autoencoder.pdf"



 75%|███████▍  | 89/119 [12:07<02:05,  4.18s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/1911.03867


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/1911.03867_A_Modular_Deep_Learning_Pipeline_for_Galaxy-Scale_Strong_Gravitational_Lens_Detection_and_Modeling.pdf"



 76%|███████▌  | 90/119 [12:12<02:00,  4.16s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2004.13048


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2004.13048_HOLISMOKES_--_II_Identifying_galaxy-scale_strong_gravitational_lenses_in_Pan-STARRS_using_convolutional_neural_networks.pdf"



 76%|███████▋  | 91/119 [12:16<01:56,  4.16s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2005.04730


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2005.04730_Discovering_New_Strong_Gravitational_Lenses_in_the_DESI_Legacy_Imaging_Surveys.pdf"



 77%|███████▋  | 92/119 [12:20<01:51,  4.12s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2005.05353


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2005.05353_Dark_Matter_Subhalos_Strong_Lensing_and_Machine_Learning.pdf"



 78%|███████▊  | 93/119 [12:24<01:47,  4.15s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2007.00188


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2007.00188_Deep_Learning_for_Strong_Lensing_Search_Tests_of_the_Convolutional_Neural_Networks_and_New_Candidates_from_KiDS_DR3.pdf"



 79%|███████▉  | 94/119 [12:28<01:43,  4.12s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2008.12731


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2008.12731_Decoding_Dark_Matter_Substructure_without_Supervision.pdf"



 80%|███████▉  | 95/119 [12:32<01:39,  4.13s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2009.06639


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2009.06639_Extracting_the_Subhalo_Mass_Function_from_Strong_Lens_Images_with_Image_Segmentation.pdf"



 81%|████████  | 96/119 [12:36<01:34,  4.11s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2009.06663


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2009.06663_Image_segmentation_for_analyzing_galaxy-galaxy_strong_lensing_systems.pdf"



 82%|████████▏ | 97/119 [12:40<01:29,  4.07s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2010.12960


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2010.12960_Hunting_for_Dark_Matter_Subhalos_in_Strong_Gravitational_Lensing_with_Neural_Networks.pdf"



 82%|████████▏ | 98/119 [12:44<01:24,  4.04s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2010.07032


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2010.07032_Targeted_Likelihood-Free_Inference_of_Dark_Matter_Substructure_in_Strongly-Lensed_Galaxies.pdf"



 83%|████████▎ | 99/119 [12:48<01:21,  4.06s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2012.00042


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2012.00042_Large-Scale_Gravitational_Lens_Modeling_with_Bayesian_Neural_Networks_for_Accurate_and_Precise_Inference_of_the_Hubble_Constant.pdf"



 84%|████████▍ | 100/119 [12:52<01:17,  4.06s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2109.00014


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2109.00014_Strong_lens_systems_search_in_the_Dark_Energy_Survey_using_Convolutional_Neural_Networks.pdf"



 85%|████████▍ | 101/119 [12:57<01:13,  4.11s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2109.09781


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2109.09781_Finding_quadruply_imaged_quasars_with_machine_learning_I_Methods.pdf"



 86%|████████▌ | 102/119 [13:01<01:09,  4.09s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.00023


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2110.00023_Mining_for_Strong_Gravitational_Lenses_with_Self-supervised_Learning.pdf"



 87%|████████▋ | 103/119 [13:05<01:05,  4.12s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.01905


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2110.01905_High-quality_strong_lens_candidates_in_the_final_Kilo_Degree_survey_footprint.pdf"



 87%|████████▋ | 104/119 [13:09<01:02,  4.15s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.02418


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2110.02418_The_DES_Bright_Arcs_Survey_Candidate_Strongly_Lensed_Galaxy_Systems_from_the_Dark_Energy_Survey_5000_Sq_Deg_Footprint.pdf"



 88%|████████▊ | 105/119 [13:13<00:58,  4.15s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.09202


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2110.09202_Finding_Strong_Gravitational_Lenses_Through_Self-Attention.pdf"



 89%|████████▉ | 106/119 [13:17<00:53,  4.15s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2110.11972


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2110.11972_Strong_lensing_in_UNIONS_Toward_a_pipeline_from_discovery_to_modeling.pdf"



 90%|████████▉ | 107/119 [13:21<00:49,  4.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2112.02479


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2112.02479_Exploring_the_interpretability_of_deep_neural_networks_used_for_gravitational_lens_finding_with_a_sensitivity_probe.pdf"



 91%|█████████ | 108/119 [13:26<00:45,  4.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2202.12776


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2202.12776_Detecting_gravitational_lenses_using_machine_learning_exploring_interpretability_and_sensitivity_to_rare_lensing_configurations.pdf"



 92%|█████████▏| 109/119 [13:30<00:41,  4.15s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2202.07663


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2202.07663_GIGA-Lens_Fast_Bayesian_Inference_for_Strong_Gravitational_Lens_Modeling.pdf"



 92%|█████████▏| 110/119 [13:34<00:37,  4.13s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2203.09536


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2203.09536_Developing_a_Victorious_Strategy_to_the_Second_Strong_Gravitational_Lensing_Data_Challenge.pdf"



 93%|█████████▎| 111/119 [13:38<00:33,  4.17s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2204.05924


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2204.05924_DeepZipper_II_Searching_for_Lensed_Supernovae_in_Dark_Energy_Survey_Data_with_Deep_Learning.pdf"



 94%|█████████▍| 112/119 [13:42<00:28,  4.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2206.02764


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2206.02764_New_Strong_Gravitational_Lenses_from_the_DESI_Legacy_Imaging_Surveys_Data_Release_9.pdf"



 95%|█████████▍| 113/119 [13:46<00:24,  4.15s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2205.09126


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2205.09126_Estimating_the_warm_dark_matter_mass_from_strong_lensing_images_with_truncated_marginal_neural_ratio_estimation.pdf"



 96%|█████████▌| 114/119 [13:50<00:20,  4.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2206.11279


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2206.11279_HOLISMOKES_--_IX_Neural_network_inference_of_strong-lens_parameters_and_uncertainties_from_ground-based_images.pdf"



 97%|█████████▋| 115/119 [13:55<00:16,  4.12s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2206.14820


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2206.14820_Strong_Lensing_Source_Reconstruction_Using_Continuous_Neural_Fields.pdf"



 97%|█████████▋| 116/119 [13:59<00:12,  4.13s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.01073


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2207.01073_Pixelated_Reconstruction_of_Gravitational_Lenses_using_Recurrent_Inference_Machines.pdf"



 98%|█████████▊| 117/119 [14:03<00:08,  4.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.04123


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2207.04123_Population-Level_Inference_of_Strong_Gravitational_Lenses_with_Neural_Network-Based_Selection_Correction.pdf"



 99%|█████████▉| 118/119 [14:07<00:04,  4.14s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.10124


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2207.10124_HOLISMOKES_--_X_Comparison_between_neural_network_and_semi-automated_traditional_modeling_of_strong_lenses.pdf"



100%|██████████| 119/119 [14:11<00:00,  7.16s/it]

Output: [arxiv-dl] (version: 1.1.5)
[1/1] >>> https://arxiv.org/abs/2207.10698


Error: [Processing] Retrieving paper metadata...
[Done] Paper PDF already exists at: "/Users/panchamb/Downloads/ArXiv_Papers/2207.10698_A_machine_learning_based_approach_to_gravitational_lens_identification_with_the_International_LOFAR_Telescope.pdf"

